### PR TITLE
Guide towards scripting instead of installers for SqlPersitence

### PR DESCRIPTION
**Pull request by andreasohlund**
Based on a discussion with @SzymonPobiega

### CODE REVIEW

File: persistence/sql/install.md [hunk 1]
```diff
@@ -7,21 +7,21 @@ reviewed: 2017-10-29
...
 
-NOTE: Note that this is also a valid approach for higher level environments.
+NOTE: Note this is only recommended during development.
 
 
-## Optionally take control of script execution
+## Script execution in higher enviroments
 
-However in higher level environment scenarios, where standard installers are being run, but the SQL persistence installation scripts have been executed as part of a deployment, it may be necessary to explicitly disable the SQL persistence installers executing while leaving standard installers enabled.
+In non-development environments, where the SQL persistence installation scripts have been executed as part of the deployment, it may be necessary to explicitly disable the SQL persistence installers if [standard installers](/nservicebus/operations/installers.md) needs to be used for other purposes.
```
Comment (ramonsmits) [inline persistence/sql/install.md]: Not sure what is meant with "if standard installers needs to be used for other purposes." what would those be? Queue creation and/or routing topology?

File: persistence/sql/installer-workflow.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ A typical workflow on developers machine consists of the following steps:
 
 NOTE: "Higher Environment" is a general term to refer to any non-development environment. For example "Integration" or "Production".
 
-The workflow in a higher environment will differ based on the specifics of an organizations process. For example, it's possible to allow endpoints [to automatically execute scripts](/persistence/sql/install.md#script-execution-runs-by-default-at-endpoint-startup) or [to take full control and execute them using custom code](/persistence/sql/install.md#optionally-take-control-of-script-execution).
+The workflow in a higher environment will differ based on the specifics of an organizations process. For example, it's possible to allow endpoints [to automatically execute scripts](/persistence/sql/install.md#script-execution-runs-by-default-at-endpoint-startup) or [to take full control and execute them using custom code](/persistence/sql/install.md#script-execution-in-higher-enviroments).
```
Comment (ramonsmits) [inline persistence/sql/installer-workflow.md]: `script-execution-in-higher-enviroments` has a typo: `enviroments`

File: persistence/sql/install.md [hunk 3]
```diff
@@ -7,21 +7,21 @@ reviewed: 2017-10-29
...
+To streamline development SQL persistence will execute generated scripts at endpoint startup if installers are enbled.
 
 snippet: ExecuteAtStartup
 
-NOTE: Note that this is also a valid approach for higher level environments.
+NOTE: Note this is only recommended during development.
 
 
-## Optionally take control of script execution
+## Script execution in higher enviroments
```
Comment (ramonsmits) [inline persistence/sql/install.md]: Typo: enviroments
```suggestion
## Script execution in higher environments
```

File: persistence/sql/install.md (hunk 1 above)
Comment (andreasohlund) [reply persistence/sql/install.md]: Yea, queue creation, gateway URLACL's etc.

Can we make this clearer?

**Pull request merged**